### ISSUE TITLE
The limit on the number of applications in the list.

### ISSUE THREAD

**Issue by dekhtyarev** [olalonde/deisdash#4]
Hello!

Thanks for a great app!
I would like to suggest an improvement: specify the limit of applications that will be displayed on the main page.
Because being the administrator of more than 100 applications - it is impossible to see all apps.

Unfortunately, I'm not good at React, Redux and Webpack, so I can not quickly implement this feature.

**Comment by olalonde** [comment]
Agreed. I think there should be some pagination or we could only display apps that are "favorited". I'm really busy at the moment so don't have time to implement this but I'll keep in mind.
